_docs/_latest/setup/upgrade.md

```diff
@@ -401,6 +401,100 @@ Forseti is updated to be usable on a non organization resource.
 {% endcapture %}
 {% include site/zippy/item.html title="Upgrading 2.3.0 to 2.4.0" content=upgrading_2_3_0_to_2_4_0 uid=5 %}
 
+{% capture upgrading_2_4_0_to_2_5_0 %}
+
+Starting v2.5, Forseti Inventory will be integrated with the Cloud 
+Asset Inventory (CAI) service if the user is deploying Forseti. CAI
```

Comment:
change 'if the user is deploying Forseti' to 'when Forseti is first deployed'

Reply:
Changed.